rh/forms.py

```diff
@@ -12,10 +14,11 @@
 FIELD_SCHEMA_VERSION = 4
 
 
-class RhForm(AddressAndBoroughFormMixin, forms.Form):
-    first_name = forms.CharField(max_length=30)
-    last_name = forms.CharField(max_length=150)
-    apartment_number = forms.CharField(max_length=15)
+class RhForm(AddressAndBoroughFormMixin, forms.ModelForm):
+    class Meta:
+        model = RentalHistoryRequest
+        fields = ('first_name', 'last_name', 'apartment_number',)
```

Comment:
@sraby, this [`forms.ModelForm`](https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/) basically makes it easy to build a form from a Django Model, so that the form's fields (and their constraints like max character count, validation logic, etc) are just "inherited" from a model.  In this case we don't need to worry about the address and borough because those fields are provided by the `AddressAndBoroughFormMixin`, and we manually specify a `phone_number` form field below, leaving just the first/last name and apartment number to take from the model.